postgres/datadog_checks/postgres/statements.py

```diff
@@ -26,6 +26,10 @@
 except ImportError:
     from ..stubs import datadog_agent
 
+QUERY_COUNTS_QUERY = """
+SELECT queryid, calls from {pg_stat_statements_view}
```

Comment:
Since you don't use the query string, you should use `pg_stat_statements(false)` to allow pgss to skip reading the query text file which can be huge and costly.